BlueprintUICommonControls/Sources/AccessibilityBlocker.swift

```diff
@@ -19,7 +19,7 @@ public struct AccessibilityBlocker: Element {
     //
 
     public var content: ElementContent {
-        return ElementContent(child: wrapped)
+        wrapped.content
```

Comment:
Need to double check: Does this work as expected with a custom backingViewDescription?

Reply:
Yeah i dont think this works, gotta revert

Reply:
Reverted – would be good to fix this, but it can be a follow up.